CODE_OF_CONDUCT.md

```diff
@@ -1,63 +0,0 @@
-# Contributor Covenant Code of Conduct
-
-## Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as
```

Comment:
This isn't really a community project, so it's hard to see why we need a code of conduct.

Reply:
Instead of adding this to `.distignore`, I thought it'd be easier to delete it.